lib/std/Build/Step/WriteFile.zig

```diff
@@ -89,18 +79,35 @@ pub fn create(owner: *std.Build) *WriteFile {
     return write_file;
 }
 
+pub fn getFile(write_file: *WriteFile, index: usize) std.Build.LazyPath {
+    if (index >= write_file.files.items.len) std.debug.panic(
+        "file index {} is too big for only having {} file(s)",
+        .{ index, write_file.files.items.len },
+    );
```

Comment:
This panic is so close to what you get with the regular out of bounds panic on line 90 below, that I think it is borderline redundant, and should be removed.